app/main/service/trip_service.py

```diff
@@ -0,0 +1,17 @@
+from app.main import db
+from app.main.model.trip import Trip
+from app.main.util.exception.TripException import TripAlreadyExistsException
+
+
+def create_trip(trip):
+    existing_trip = Trip.query.filter_by(creator_id=trip.creator_id).filter_by(name=trip.name).first()
```

Comment:
.first() si on a pas ce trip, on obtient quoi ?

Reply:
`None`, d'où le `if not existing_trip` sous cette ligne